eradiate/scenes/biosphere.py

```diff
@@ -325,6 +332,22 @@ class HomogeneousDiscreteCanopy(SceneElement):
         ]
     )
 
+    leaf_positions = attrib_quantity(
+        default=ureg.Quantity(np.array([[0, 0, 0]]), ureg.m),
+        units_compatible=cdu.generator("length"),
+    )
+
+    leaf_orientations = attrib_quantity(
```

Comment:
Since this is meant to be unitless, you can actually declare it as a regular `attr.ib`: you don't need the extra bells and whistles for conversion. You'll however need to validate for consistency with the other attributes.